.github/workflows/test.yml

```diff
@@ -9,11 +9,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [16, 18]
+        node: [18, 20]
```

Comment:
Looks good now!  We can add 22 in a couple months once its part of the official releases.  Lets also maintain 18 until end of life.